CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.11.3] - 2023-08-02
+### Changed
+- Exclude jetty-all from hive-service in pom.xml of core module, because it makes spring start fail and makes `WaggleDanceIntegrationTest` fail.
```

Comment:
I can't really reproduce this and the tests run fine in our GH actions, can you tell what environment you are running that's causing this issue?